vnext/Microsoft.ReactNative/Modules/Animated/NativeAnimatedNodeManager.cpp

```diff
@@ -120,10 +120,37 @@ void NativeAnimatedNodeManager::DisconnectAnimatedNode(int64_t parentNodeTag, in
   }
 }
 
-void NativeAnimatedNodeManager::StopAnimation(int64_t animationId) {
+void NativeAnimatedNodeManager::StopAnimation(int64_t animationId, bool isTrackingAnimation) {
   if (m_activeAnimations.count(animationId)) {
-    if (const auto animation = m_activeAnimations.at(animationId).get()) {
-      animation->StopAnimation();
+    if (const auto animation = m_activeAnimations.at(animationId)) {
```

Comment:
Not sure why `.get()` was called, perhaps we should add it back...

Reply:
I think it's just extracting the raw pointer to the COM object instead of something that will refcount. The smart pointer still has a boolean operatio, so the `get()` is fine to remove.

Reply:
Needed to remove the .get() anyway because we need to persist the shared pointer in the pendingCompletionAnimation map until its completion callback fires.